.github/workflows/ci.yml

```diff
@@ -7,79 +7,44 @@ on:
 name: Continuous integration
 
 jobs:
-  ubuntu-build-x86_64:
-    name: Ubuntu x86_64
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          submodules: recursive
-      - name: Cache CMake build directory
-        uses: actions/cache@v4
-        with:
-          path: |
-            build
-          key: ${{ runner.os }}-${{ runner.arch }}-cmake-build-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-${{ runner.arch }}-cmake-build-
-      - name: Install dependencies
-        run: sudo apt-get update && sudo apt-get install -y
-          libasound2-dev
-          libgl-dev
-          libglu1-mesa-dev
-          libgtk-3-dev
-          libjack-dev
-          libmad0-dev
-          libpulse-dev
-          libudev-dev
-          libxinerama-dev
-          libx11-dev
-          libxrandr-dev
-          libxtst-dev
-          nasm
-      - name: Configure
-        run: cmake -B build -DWITH_FFMPEG_JOBS="$(nproc)"
-      - name: Build
-        run: cmake --build build --parallel "$(nproc)"
-
-  ubuntu-build-arm:
-    name: Ubuntu ARM
-    runs-on: ubuntu-24.04-arm
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          submodules: recursive
-      - name: Cache CMake build directory
-        uses: actions/cache@v4
-        with:
-          path: |
-            build
-          key: ${{ runner.os }}-${{ runner.arch }}-cmake-build-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-${{ runner.arch }}-cmake-build-
-      - name: Install dependencies
-        run: sudo apt-get update && sudo apt-get install -y
-          libasound2-dev
-          libgl-dev
-          libglu1-mesa-dev
-          libgtk-3-dev
-          libjack-dev
-          libmad0-dev
-          libpulse-dev
-          libudev-dev
-          libxinerama-dev
-          libx11-dev
-          libxrandr-dev
-          libxtst-dev
-          nasm
-      - name: Configure
-        run: cmake -B build -DWITH_FFMPEG_JOBS="$(nproc)" -DWITH_MINIMAID=OFF
-      - name: Build
-        run: cmake --build build --parallel "$(nproc)"
-
-  macos-build-arm64:
-    name: macOS (M1)
-    runs-on: macos-latest
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, ubuntu-24.04-arm, macos-latest, macos-13, windows-latest]
+        include:
+          - os: ubuntu-latest
+            name: Ubuntu x86_64
```

Comment:
the names don't seem to work, all jobs generated from the matrix are called `build (<os value>)`. If it's impossible to fix then I suppose we don't need to define them at all?

Reply:
Good point, corrected with https://github.com/itgmania/itgmania/pull/671/commits/10beb4e7948d7c1e638abd34f85784a6b9b1b196